test/y2security/lsm/config_test.rb

```diff
@@ -258,4 +258,29 @@
       end
     end
   end
+
+  describe "#export" do
+    context "when there is no LSM selected" do
+      it "returns an empty hash" do
+        expect(subject.export).to eql({})
+      end
+    end
+
+    context "when a LSM is selected" do
+      it "exports the selected LSM" do
+        subject.select("apparmor")
+        expect(subject.export).to eql("lsm_select" => "apparmor")
+      end
+
+      context "and it is SELinux" do
+        it "exports the SELInux mode" do
```

Comment:
NP: SEL**i**nux

Reply:
I would say "exports also the SELinux mode" to make it obvious that selected lsm is also exported.